storybook/src/components/button/button.css-template.ts

```diff
@@ -70,7 +71,7 @@ export const cssButton: ComponentImplementation<Button | ButtonAnchor> = {
         <button
           type=${type}
           id=${ifDefined(id || undefined)}
-          class=${ifDefined(className || undefined)}
+          class=${ifDefined(`${className} ${compact ? "btn-sm" : ""}`.trim() || undefined)}
```

Comment:
Ik vind [classname, compact ? 'btn-sm' : ''].join(' ') || undefined` ook altijd wel fraai.

Overigens heeft Lit ook een `classMap` directive, maar die maakt gruwelijk lelijke markup (`class="  "` bij leeg object, of `class=" koe konijn "` 😱 ).

Reply:
Ik weet niet hoe je dit `[classname, compact ? 'btn-sm' : ''].join(' ') || undefined` doet, maar bij mij dit doet hij het niet 🔴

Reply:
Solved!

Reply:
Zal mijn typefout wel zijn geweest `classname` vs `className`. Maar mooi! Het was overigens gewoon 'n suggestie, dit soort constructies zijn eigenlijk altijd wel vreselijk.

Ik vind `clsx()` de mooiste oplossing en ik bedenk mij net dat we die misschien ook wel in Lit templates kunnen gebruiken: `class=${ifDefined(clsx([className, { 'btn-sm': !!compact }]) || undefined)}`. Maar laten we daar maar 'n andere keer naar kijken, ik weet niet of deze package al een dependency heeft op `clsx`.

Anyways, lekker bezig.